meta-mender-nxp/README.md

```diff
@@ -2,9 +2,9 @@
 
 Supported boards:
 
- - WaRP7
```

Comment:
Sorry I missed this, but could you update the title of the README.md to 

```
# meta-mender-nxp
```

And then add a short description as well.

Reply:
Done ;)